src/pages/home/components/countries/styles.scss

```diff
@@ -0,0 +1,58 @@
+@import '~styles/settings';
+
+.c-choose-country {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .row{
+    margin: 50px 0;
+
+    &>div{
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  .shape {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+  }
+
+  a {
+    position: relative;
+
+    &:hover .shape__no-hover {
+      opacity: 0;
+      pointer-events: all;
+      transform: scale3d(0.8, 0.8, 1);
+    }
+
+    &:hover .shape__hover {
+      opacity: 1;
+      transform: scale3d(1, 1, 1);
+    }
+
+    .shape__hover {
+      position: absolute;
+      top: -20%;
+      left: 0;
+      opacity: 0;
+      pointer-events: none;
+      transition: all 0.05s $ease-in-out-sine;
+      transform: scale3d(0.8, 0.8, 1);
+      transform-origin: left;
+    }
+
+    .shape__no-hover {
+      opacity: 1;
+      transition: all 0.05s $ease-in-out-sine;
+      transform: scale3d(1, 1, 1);
+      transform-origin: left;
+    }
+  }
+}
```

Comment:
you can remove that file as it is not used anywhere (that's a leftover from merge)